client/src/main/java/crawlercommons/urlfrontier/client/GetURLStatus.java

```diff
@@ -0,0 +1,101 @@
+package crawlercommons.urlfrontier.client;
+
+import crawlercommons.urlfrontier.URLFrontierGrpc;
+import crawlercommons.urlfrontier.URLFrontierGrpc.URLFrontierBlockingStub;
+import crawlercommons.urlfrontier.Urlfrontier.URLItem;
+import crawlercommons.urlfrontier.Urlfrontier.URLStatusRequest;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import picocli.CommandLine.ArgGroup;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.ParentCommand;
+
+@Command(name = "GetURLStatus", description = "Get the status of an URL")
+public class GetURLStatus implements Runnable {
+
+    @ParentCommand private Client parent;
+
+    @ArgGroup(exclusive = false, multiplicity = "1")
+    Dependent group;
+
+    static class Dependent {
+        @Option(
+                names = {"-c", "--crawlID"},
```

Comment:
The crawlID when not specified by the client is replaced by **DEFAULT** on the server side. This should be optional and the behaviour should be the same when calling getURLStatus as it is when calling the other endpoints

Reply:
Changed